Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/.idea/misc.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -17,6 +17,9 @@
         <entry key="..\:/AndroidStudioProjects/HR/app/src/main/res/layout/fragment_register_container.xml" value="0.275" />
         <entry key="..\:/AndroidStudioProjects/HR/app/src/main/res/layout/item_employee.xml" value="0.275" />
         <entry key="..\:/AndroidStudioProjects/HR/app/src/main/res/layout/item_faq.xml" value="0.275" />
+        <entry key="..\:/AndroidStudioProjects/HR/app/src/main/res/layout/item_order.xml" value="0.3" />
+        <entry key="..\:/AndroidStudioProjects/HR/app/src/main/res/layout/item_order_filter.xml" value="0.275" />
+        <entry key="..\:/AndroidStudioProjects/HR/app/src/main/res/layout/item_orders_filter.xml" value="0.275" />
         <entry key="..\:/AndroidStudioProjects/HR/app/src/main/res/menu/bottom_nav_menu.xml" value="0.275" />
       </map>
     </option>
Index: app/src/main/java/com/tinkoff/hr/data/Filter.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/data/Filter.kt b/app/src/main/java/com/tinkoff/hr/data/Filter.kt
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/java/com/tinkoff/hr/data/Filter.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,6 @@
+package com.tinkoff.hr.data
+
+data class Filter(
+    val name: String,
+    var isChecked: Boolean = false
+)
\ No newline at end of file
Index: app/src/main/java/com/tinkoff/hr/data/Order.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/data/Order.kt b/app/src/main/java/com/tinkoff/hr/data/Order.kt
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/java/com/tinkoff/hr/data/Order.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,10 @@
+package com.tinkoff.hr.data
+
+data class Order(
+    val id: Int,
+    val article: String,
+    val name: String,
+    val count: Int,
+    var selected: Boolean,
+    val photoUrl: String? = null
+)
\ No newline at end of file
Index: app/src/main/java/com/tinkoff/hr/view/employees/EmployeeDescFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/view/employees/EmployeeDescFragment.kt b/app/src/main/java/com/tinkoff/hr/view/employees/EmployeeDescFragment.kt
--- a/app/src/main/java/com/tinkoff/hr/view/employees/EmployeeDescFragment.kt	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/java/com/tinkoff/hr/view/employees/EmployeeDescFragment.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -27,7 +27,6 @@
         setEmployee(binding)
         setEditableListeners(binding)
 
-
         return binding.root
     }
 
Index: app/src/main/java/com/tinkoff/hr/view/employees/EmployeesFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/view/employees/EmployeesFragment.kt b/app/src/main/java/com/tinkoff/hr/view/employees/EmployeesFragment.kt
--- a/app/src/main/java/com/tinkoff/hr/view/employees/EmployeesFragment.kt	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/java/com/tinkoff/hr/view/employees/EmployeesFragment.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -21,7 +21,6 @@
 
         val employee = Employee("this_is_employee@gmail.com", "Иванов Александр Александрович")
 
-        binding.rvEmployees.layoutManager = LinearLayoutManager(requireContext())
         binding.rvEmployees.adapter = EmployeesAdapter(
             listOf(
                 employee,
Index: app/src/main/java/com/tinkoff/hr/view/faq/FAQAdapter.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/view/faq/FAQAdapter.kt b/app/src/main/java/com/tinkoff/hr/view/faq/FAQAdapter.kt
--- a/app/src/main/java/com/tinkoff/hr/view/faq/FAQAdapter.kt	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/java/com/tinkoff/hr/view/faq/FAQAdapter.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -17,7 +17,6 @@
     class ViewHolder(val binding: ItemFaqBinding) : RecyclerView.ViewHolder(binding.root) {
         fun bind(faq: FAQ){
             binding.faq = faq
-            binding.executePendingBindings()
         }
     }
 
Index: app/src/main/java/com/tinkoff/hr/view/faq/FAQFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/view/faq/FAQFragment.kt b/app/src/main/java/com/tinkoff/hr/view/faq/FAQFragment.kt
--- a/app/src/main/java/com/tinkoff/hr/view/faq/FAQFragment.kt	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/java/com/tinkoff/hr/view/faq/FAQFragment.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -26,7 +26,6 @@
                     "Если есть клиника, в которую ты ходишь, но ее нет в переченье, то пиши сюда tcr_dms@tinkoff.ru;"
         )
 
-        binding.rvFaq.layoutManager = LinearLayoutManager(requireContext())
         binding.rvFaq.adapter = FAQAdapter(
             listOf(faq, faq, faq, faq, faq, faq, faq, faq, faq, faq, faq)
         )
Index: app/src/main/java/com/tinkoff/hr/view/orders/FiltersAdapter.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/view/orders/FiltersAdapter.kt b/app/src/main/java/com/tinkoff/hr/view/orders/FiltersAdapter.kt
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/java/com/tinkoff/hr/view/orders/FiltersAdapter.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,43 @@
+package com.tinkoff.hr.view.orders
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.tinkoff.hr.R
+import com.tinkoff.hr.data.Filter
+import com.tinkoff.hr.databinding.ItemOrdersFilterBinding
+
+class FiltersAdapter(private val data: List<Filter>, private val listener: OnFilterClickListener) : RecyclerView.Adapter<FiltersAdapter.ViewHolder>() {
+
+    class ViewHolder(val binding: ItemOrdersFilterBinding) : RecyclerView.ViewHolder(binding.root){
+        fun bind(filter: Filter){
+            binding.filter = filter
+        }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        val binding: ItemOrdersFilterBinding =
+            DataBindingUtil.inflate(inflater, R.layout.item_orders_filter, parent, false)
+        return ViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val filter = data[position]
+        holder.bind(filter)
+
+        holder.binding.chipFilter.setOnCheckedChangeListener { chip, isChecked ->
+            listener.onFilterClick(isChecked, position)
+        }
+    }
+
+    override fun getItemCount(): Int {
+        return data.size
+    }
+
+    interface OnFilterClickListener{
+        fun onFilterClick(isChecked: Boolean, position: Int)
+    }
+}
\ No newline at end of file
Index: app/src/main/java/com/tinkoff/hr/view/orders/OrdersAdapter.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/view/orders/OrdersAdapter.kt b/app/src/main/java/com/tinkoff/hr/view/orders/OrdersAdapter.kt
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/java/com/tinkoff/hr/view/orders/OrdersAdapter.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,78 @@
+package com.tinkoff.hr.view.orders
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.card.MaterialCardView
+import com.tinkoff.hr.R
+import com.tinkoff.hr.data.Order
+import com.tinkoff.hr.databinding.ItemOrderBinding
+
+class OrdersAdapter(
+    private val data: List<Order>,
+    private val listener: OnItemClickListener
+) : RecyclerView.Adapter<OrdersAdapter.ViewHolder>() {
+
+    class ViewHolder(val binding: ItemOrderBinding) : RecyclerView.ViewHolder(binding.root) {
+        fun bind(order: Order) {
+            binding.order = order
+        }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        val binding: ItemOrderBinding =
+            DataBindingUtil.inflate(inflater, R.layout.item_order, parent, false)
+        return ViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val order = data[position]
+        holder.bind(order)
+
+        setRightCardBackground(holder.binding.layoutOrder, order.selected)
+
+        holder.binding.layoutOrder.setOnClickListener {
+            val isSelected: Boolean
+            if (order.selected) {
+                isSelected = false
+                setRightCardBackground(holder.binding.layoutOrder, false)
+            } else {
+                isSelected = true
+                setRightCardBackground(holder.binding.layoutOrder, true)
+            }
+            listener.onItemClick(position, isSelected)
+        }
+    }
+
+    private fun setRightCardBackground(v: MaterialCardView, selected: Boolean){
+        if (!selected) {
+            v.elevation = 10f
+            v.setCardBackgroundColor(
+                ContextCompat.getColor(
+                    v.context,
+                    R.color.white
+                )
+            )
+        } else {
+            v.elevation = 0f
+            v.setCardBackgroundColor(
+                ContextCompat.getColor(
+                    v.context,
+                    R.color.gray
+                )
+            )
+        }
+    }
+
+    override fun getItemCount(): Int {
+        return data.size
+    }
+
+    interface OnItemClickListener {
+        fun onItemClick(position: Int, isSelected: Boolean)
+    }
+}
\ No newline at end of file
Index: app/src/main/java/com/tinkoff/hr/view/orders/OrdersFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/view/orders/OrdersFragment.kt b/app/src/main/java/com/tinkoff/hr/view/orders/OrdersFragment.kt
--- a/app/src/main/java/com/tinkoff/hr/view/orders/OrdersFragment.kt	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/java/com/tinkoff/hr/view/orders/OrdersFragment.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -5,17 +5,50 @@
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
-import com.tinkoff.hr.R
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.tinkoff.hr.data.Order
+import com.tinkoff.hr.databinding.FragmentOrdersBinding
+import com.tinkoff.hr.viewmodels.FiltersViewModel
+import com.tinkoff.hr.viewmodels.OrdersViewModel
 
-class OrdersFragment : Fragment() {
+class OrdersFragment : Fragment(), OrdersAdapter.OnItemClickListener, FiltersAdapter.OnFilterClickListener {
+
+    private val filtersViewModel: FiltersViewModel by viewModels()
+    private val ordersViewModel: OrdersViewModel by viewModels()
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        val view = inflater.inflate(R.layout.fragment_orders, container, false)
+        val binding = FragmentOrdersBinding.inflate(inflater, container, false)
+
+        binding.rvOrders.layoutManager = GridLayoutManager(requireContext(), 2)
+
+        ordersViewModel.getOrders().observe(viewLifecycleOwner){
+            val ordersAdapter = OrdersAdapter(it, this)
+            binding.rvOrders.adapter = ordersAdapter
+        }
+
 
-        return view
+        binding.rvFilters.layoutManager =
+            LinearLayoutManager(requireContext(), LinearLayoutManager.HORIZONTAL, false)
+
+        filtersViewModel.getFilters().observe(viewLifecycleOwner){
+            val filtersAdapter = FiltersAdapter(it, this)
+            binding.rvFilters.adapter = filtersAdapter
+        }
+
+        return binding.root
+    }
+
+    override fun onFilterClick(isChecked: Boolean, position: Int) {
+        filtersViewModel.setFilterIsChecked(isChecked, position)
+    }
+
+    override fun onItemClick(position: Int, isSelected: Boolean) {
+        ordersViewModel.setIsSelected(position, isSelected)
     }
 }
\ No newline at end of file
Index: app/src/main/java/com/tinkoff/hr/viewmodels/FiltersViewModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/viewmodels/FiltersViewModel.kt b/app/src/main/java/com/tinkoff/hr/viewmodels/FiltersViewModel.kt
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/java/com/tinkoff/hr/viewmodels/FiltersViewModel.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,35 @@
+package com.tinkoff.hr.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.tinkoff.hr.data.Filter
+
+class FiltersViewModel : ViewModel() {
+    private val filters: MutableLiveData<List<Filter>> by lazy {
+        MutableLiveData<List<Filter>>().also {
+            it.value = listOf(
+                Filter("Всё"),
+                Filter("Бакалея"),
+                Filter("Молочные продукты"),
+                Filter("Печеньки"),
+                Filter("Кофе"),
+                Filter("Выпечка"),
+                Filter("Всё"),
+                Filter("Бакалея"),
+                Filter("Молочные продукты"),
+                Filter("Печеньки"),
+                Filter("Кофе"),
+                Filter("Выпечка")
+            )
+        }
+    }
+
+    fun getFilters(): LiveData<List<Filter>> {
+        return filters
+    }
+
+    fun setFilterIsChecked(isChecked: Boolean, position: Int) {
+        filters.value?.get(position)?.isChecked = isChecked
+    }
+}
\ No newline at end of file
Index: app/src/main/java/com/tinkoff/hr/viewmodels/OrdersViewModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/tinkoff/hr/viewmodels/OrdersViewModel.kt b/app/src/main/java/com/tinkoff/hr/viewmodels/OrdersViewModel.kt
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/java/com/tinkoff/hr/viewmodels/OrdersViewModel.kt	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,30 @@
+package com.tinkoff.hr.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.tinkoff.hr.data.Order
+
+class OrdersViewModel : ViewModel() {
+    private val orders: MutableLiveData<List<Order>> by lazy {
+        MutableLiveData<List<Order>>().also {
+            it.value = listOf(
+                Order(1, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, false),
+                Order(2, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, true),
+                Order(3, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, false),
+                Order(1, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, false),
+                Order(2, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, true),
+                Order(3, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, false),
+                Order(1, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, false),
+                Order(2, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, true),
+                Order(3, "", "Сушки FINE LIFE Малютка с маком, 350 г", 199, false),
+            )
+        }
+    }
+
+    fun getOrders(): LiveData<List<Order>> = orders
+
+    fun setIsSelected(position: Int, isSelected: Boolean){
+        orders.value?.get(position)?.selected = isSelected
+    }
+}
\ No newline at end of file
Index: app/src/main/res/layout/fragment_employees.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_employees.xml b/app/src/main/res/layout/fragment_employees.xml
--- a/app/src/main/res/layout/fragment_employees.xml	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/res/layout/fragment_employees.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -10,6 +10,7 @@
             android:id="@+id/rv_employees"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
Index: app/src/main/res/layout/fragment_faq.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_faq.xml b/app/src/main/res/layout/fragment_faq.xml
--- a/app/src/main/res/layout/fragment_faq.xml	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/res/layout/fragment_faq.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -4,8 +4,7 @@
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/white">
+        android:layout_height="match_parent">
 
         <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/rv_faq"
@@ -14,7 +13,8 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintTop_toTopOf="parent"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"/>
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 </layout>
\ No newline at end of file
Index: app/src/main/res/layout/fragment_orders.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_orders.xml b/app/src/main/res/layout/fragment_orders.xml
--- a/app/src/main/res/layout/fragment_orders.xml	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/res/layout/fragment_orders.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -2,19 +2,37 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/white">
+        android:fitsSystemWindows="true">
 
-        <TextView
-            android:layout_width="wrap_content"
+        <com.google.android.material.appbar.AppBarLayout
+            android:id="@+id/appBarLayout"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/orders"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            android:background="@color/transparent">
+
 
-    </androidx.constraintlayout.widget.ConstraintLayout>
+            <com.google.android.material.appbar.CollapsingToolbarLayout
+                android:layout_width="match_parent"
+                android:layout_height="50dp"
+                android:background="@color/white"
+                app:layout_scrollFlags="scroll|enterAlways">
+
+                <androidx.recyclerview.widget.RecyclerView
+                    android:id="@+id/rv_filters"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"/>
+
+            </com.google.android.material.appbar.CollapsingToolbarLayout>
+        </com.google.android.material.appbar.AppBarLayout>
+
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/rv_orders"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:layout_behavior="com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior" />
+    </androidx.coordinatorlayout.widget.CoordinatorLayout>
 </layout>
\ No newline at end of file
Index: app/src/main/res/layout/item_faq.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/item_faq.xml b/app/src/main/res/layout/item_faq.xml
--- a/app/src/main/res/layout/item_faq.xml	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/res/layout/item_faq.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -15,13 +15,9 @@
         <com.google.android.material.card.MaterialCardView
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:layout_marginStart="8dp"
-            android:layout_marginTop="4dp"
-            android:layout_marginEnd="8dp"
-            android:layout_marginBottom="8dp"
-            app:cardBackgroundColor="@color/gray"
+            android:layout_margin="8dp"
             app:cardCornerRadius="16dp"
-            app:cardElevation="6dp"
+            app:cardBackgroundColor="@color/gray"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
Index: app/src/main/res/layout/item_order.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/item_order.xml b/app/src/main/res/layout/item_order.xml
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/res/layout/item_order.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="order"
+            type="com.tinkoff.hr.data.Order" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <com.google.android.material.card.MaterialCardView
+            android:id="@+id/layout_order"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="8dp"
+            app:cardCornerRadius="16dp"
+            app:cardElevation="6dp"
+            android:background="?android:attr/selectableItemBackground"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent">
+
+                <com.google.android.material.imageview.ShapeableImageView
+                    android:id="@+id/shapeableImageView2"
+                    android:layout_width="150dp"
+                    android:layout_height="150dp"
+                    android:src="@drawable/ic_launcher_background"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <TextView
+                    android:id="@+id/tv_product_name"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="16dp"
+                    android:layout_marginTop="8dp"
+                    android:gravity="center"
+                    android:text="@{order.name}"
+                    android:textStyle="bold"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/shapeableImageView2" />
+
+                <TextView
+                    android:id="@+id/tv_count"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="8dp"
+                    android:layout_marginTop="8dp"
+                    android:text="@{`Кол-во ` + order.count}"
+                    app:layout_constraintTop_toBottomOf="@id/tv_product_name"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </com.google.android.material.card.MaterialCardView>
+    </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
\ No newline at end of file
Index: app/src/main/res/layout/item_orders_filter.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/item_orders_filter.xml b/app/src/main/res/layout/item_orders_filter.xml
new file mode 100644
--- /dev/null	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
+++ b/app/src/main/res/layout/item_orders_filter.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+        <variable
+            name="filter"
+            type="com.tinkoff.hr.data.Filter" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="wrap_content"
+        android:layout_height="50dp">
+
+        <com.google.android.material.chip.Chip
+            android:id="@+id/chip_filter"
+            style="@style/Widget.MaterialComponents.Chip.Choice"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:checked="@{filter.checked}"
+            android:text="@{filter.name}"
+            android:layout_margin="4dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
\ No newline at end of file
Index: app/src/main/res/values/colors.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/values/colors.xml b/app/src/main/res/values/colors.xml
--- a/app/src/main/res/values/colors.xml	(revision de3462b5e0926f3454a4de77de917bdcac780486)
+++ b/app/src/main/res/values/colors.xml	(revision 703460513ac0f67bf3685f5ffbf4a51e439dd188)
@@ -8,5 +8,5 @@
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
     <color name="transparent">#00FFFFFF</color>
-    <color name="gray">#EEEEEE</color>
+    <color name="gray">#EFEFEF</color>
 </resources>
\ No newline at end of file
